zoho/books/client.py

```diff
@@ -44,3 +44,15 @@ def get_records(self, module_name):
             response = self._get(url, params=dict({'page': int(page) + 1}, **base_params))
             all_data.extend(response[module_conf['list_name']])
         return all_data
+
+    def get_specific_record(self, module, id):
+        """
+        :return:
+        """
+        endpoint = '{0}/{1}'.format(module, id)
+        url = self.BASE_URL + str(endpoint)
```

Comment:
You should pass the `organization_id` in parameter like in `get_records`

Reply:
Good catch. Fixed